v3/pkg/application/linux_cgo.go

```diff
@@ -1131,6 +1145,12 @@ func (w *linuxWebviewWindow) setTitle(title string) {
 	}
 }
 
+func (w *linuxWebviewWindow) setIcon(icon pointer) {
+	if icon != nil {
+		C.gtk_window_set_icon(w.gtkWindow(), (*C.GdkPixbuf)(icon))
```

Comment:
Is setting a `nil` icon the method to remove the icon?  Presumably

Reply:
It is checked for `nil`, because there could go something wrong while loading the icon at `linuxApp.setIcon`. e.g. someone could set `application.Options.Icon` to an invalid image format